resources/languages/en.js

```diff
@@ -140,7 +140,7 @@ locale["services[6]"] = "HipChat is hosted group chat and video chat built for t
 locale["services[7]"] = "Telegram is a messaging app with a focus on speed and security. It’s super-fast, simple, secure and free.";
 locale["services[8]"] = "WeChat is a free messaging calling app that allows you to easily connect with family; friends across countries. It’s the all-in-one communications app for free text (SMS/MMS), voice; video calls, moments, photo sharing, and games.";
 locale["services[9]"] = "Gmail, Google's free email service, is one of the world's most popular email programs.";
-locale["services[10]"] = "Inbox by Gmail is a new app from the Gmail team. Inbox is an organized place to get things done and get back to what matters. Bundles keep emails organized.";
+locale["services[10]"] = "Google Calendar is a time-management and scheduling calendar service developed by Google.";
```

Comment:
Won't this PR also need a corresponding line for Glowing Bear in this file?

Reply:
No.